README.md

```diff
@@ -32,3 +32,116 @@ zopen build
 Note that running `zopen build` against your development repo will NOT install the code into your
 environment.
 
+
+# Developer Notes
+
+## workflow
+
+Say you want to work on meta
```

Comment:
I'd be more specific here and say: If you want to contribute to the meta repository, follow the steps below

Reply:
Yes, indeed. Thanks